discv5/discv5-wire.md

```diff
@@ -263,14 +163,14 @@ PONG is the reply to PING.
 
 ### FINDNODE Request (0x03)
 
-    message-data = [request-id, distance]
+    message-data = [request-id, distance₁, distance₂, ..., distanceₙ]
     message-type = 0x03
-    distance     = the requested log2 distance, a positive integer
+    distanceₙ    = requested log2 distance, a positive integer
+
+FINDNODE queries for nodes at the given logarithmic distances from the recipient's node ID.
+The recipient should create a result set containing nodes from its local node table.
 
-FINDNODE queries for nodes at the given logarithmic distance from the recipient's node ID.
-The node IDs of all nodes in the response must have a shared prefix length of `distance`
-with the recipient's node ID. A request with distance `0` should return the recipient's
-current record as the only result.
+A request with distance `0` should return the recipient's current record as the only result.
```

Comment:
I think the wording here leaves some ambiguity as to what the correct behavior would be for a request that includes `distance == 0` as well as other distances.  `[<request-id> , 0, 255]`

Logically, it should contain your local ENR **and** some set of ENR records at log distance 255, but the wording here suggests that the response should **only** contain the local ENR record.

Reply:
I understand your concern, will update.